.github/workflows/sanitizers.yml

```diff
@@ -15,11 +15,11 @@ jobs:
           - name: asan
             cmake-flags: "-DCMAKE_BUILD_TYPE=Asan"
             ctest-env: ""
-            ctest-flags: "-E 'ntcore|wpilibc'"
+            ctest-flags: "-E 'wpilibc'"
           - name: tsan
             cmake-flags: "-DCMAKE_BUILD_TYPE=Tsan"
             ctest-env: "TSAN_OPTIONS=second_deadlock_stack=1"
-            ctest-flags: "-E 'ntcore|cscore|cameraserver|wpilibc|wpilibNewCommands'"
```

Comment:
Don't we want the sanitizers to run on ntcore?

Reply:
The -E flag skips targets during the sanitizer runs, so removing this line actually enables tsan for ntcore.

Reply:
Ok, great